### ISSUE TITLE
ggml vicuna llama_cpp/llama.py init assert self.ctx is not None

### ISSUE THREAD

**Issue by milobestcat** [ParisNeo/lollms-webui#195]
Hey I'm trying to install ggml vicuna following this link: https://agi-sphere.com/install-textgen-webui-mac/

The texgen-webui works as I tested it on a small LLM model, however, it doesn't work for ggml models. Below is the error:

Traceback (most recent call last):
File “/Users/username/text-generation-webui/server.py”, line 67, in load_model_wrapper
shared.model, shared.tokenizer = load_model(shared.model_name)
File “/Users/username/text-generation-webui/modules/models.py”, line 85, in load_model
output = load_func(model_name)
File “/Users/username/text-generation-webui/modules/models.py”, line 242, in llamacpp_loader
model, tokenizer = LlamaCppModel.from_pretrained(model_file)
File “/Users/username/text-generation-webui/modules/llamacpp_model.py”, line 50, in from_pretrained
self.model = Llama(**params)
File “/Users/username/opt/miniconda3/envs/vicuna_ggml/lib/python3.10/site-packages/llama_cpp/llama.py”, line 161, in init
assert self.ctx is not None
AssertionError

Any idea of how to fix this?

**Comment by PoyBoi** [comment]
Did you manage to get it figured out ? @milobestcat

**Comment by YingxiaoKong** [comment]
@PoyBoi Yup, to make it work you need to install the most recent ggml model like q5_1, besides for pytorch installation, instead of using the given command, I used this one: pip install -U --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
